target/linux/x86/image/64.mk

```diff
@@ -5,3 +5,16 @@ define Device/generic
   GRUB2_VARIANT := generic
 endef
 TARGET_DEVICES += generic
+
+define Device/pcengines_apu2
+  DEVICE_TITLE := PC Engines APU 2
+  DEVICE_PACKAGES += amd64-microcode flashrom fstrim irqbalance kmod-bnx2 \
+	kmod-crypto-hw-ccp kmod-crypto-hw-ccp kmod-e1000 kmod-e1000e \
+	kmod-forcedeth kmod-gpio kmod-gpio-button-hotplug kmod-gpio-nct5104d \
+	kmod-igb kmod-leds-apu2 kmod-leds-gpio kmod-pcspkr kmod-r8169 \
+	kmod-sound-core kmod-sp5100_tco kmod-usb-core kmod-usb-ohci kmod-usb2 \
+	kmod-usb3
+  GRUB2_VARIANT := generic
+  SUPPORTED_DEVICES += pcengines,apu2
```

Comment:
Is this used/needed on x86?

Reply:
Would be if x86 store the profile in a useful way instead of using the CPU name as `board_name`. Maybe we can extend x86 so it writes the first *supported device* into the `board_name` file?

Reply:
Can help you there, I'm a x86 noob, I just was startled by the SUPPORTED_DEVICES which supposedly does not belong here.

Reply:
I guess you mean *can't*? I'd say it belongs there so the APU2 board can make sure that the correct image is installed. If not, it would lose some important functionality after an upgrade.

Reply:
> I guess you mean can't? 

Reading my thoughts ... :-)

Anyway, just wanted to highlight this; maybe just leave this open until a proper reviewer comes by ...

Reply:
So you're creating new device just because of `kmod-leds-apu2` module? Can't we just include it to the generic modules?

Reply:
Are we now going to create images for ~1k of x86/64 based devices out there? I don't think so...

Reply:
I though that APU and Alix boards are popular enough to have their own images. Based on the [wiki page](https://openwrt.org/toh/pcengines/apu2#kernel_modules) it's look like not only the LED's want some extra kernel modules. Also `sdhci-pci` seems required to support SDCard booting, I remember stumbling over this some time ago.

However as this introduces the danger of thousands of PRs adding support for exotic x86 kmod combinations, this PR can also be closed or we come up with a sane policy.

Reply:
> I though that APU and Alix boards are popular enough to have their own images. 

Yet nobody complained loud enough, that something is missing in the images yet :-)

> seems required to support SDCard booting

I'm using my apu2c sometimes for testing solely with SDCard, no issues so far.